gdeep/search/hpo.py

```diff
@@ -216,6 +217,7 @@ def __init__(
         self.scalars_dict: Dict[str, Any] = dict()
         # can be changed by changing this attribute
         self.store_pickle: bool = False
+        self.regularize = False
```

Comment:
How would I set regularize to `True` from the outside?

Reply:
done